src/onboarding/education/OnboardingEducationScreen.tsx

```diff
@@ -20,26 +21,36 @@ function useStep() {
   const { t } = useTranslation()
 
   return React.useMemo(() => {
-    return [
+    const screenSet = Math.random() < 0.5 ? 0 : 1
```

Comment:
probably a dumb question:  but why aren't we using the AB testing framework for this?

Reply:
Since it's on the initial app screens we're not guaranteed to have loaded firebase / don't want rerender the screens when firebase loads changing the order.